### DISCUSSION TITLE
FQDN support in pg_hba

### DISCUSSION THREAD

**Discussion by MathisRudolf** [pgbouncer/pgbouncer#567] [Ideas]
Hi,

we had a situations with a customer where FQDN support for pg_hba was asked. The customer thought pgbouncer is implementing pg_hba with all the features postgres does, but ran into this delta. Postgres seems to solve this with rDNS. Was there prior work in this direction? Maybe we can put some effort into it with a bit of guidance.

Kind regards
Mathis

**Comment by petere** [comment]
The pg_hba.conf implementation of PgBouncer does not match all the features in core PostgreSQL. It's just work needed that needs to be done bit by bit.
